documentation/modules/ROOT/pages/transformations/outbox-event-router.adoc

```diff
@@ -226,8 +285,8 @@ transforms.outbox.type=io.debezium.transforms.outbox.EventRouter
 value.converter=io.debezium.converters.BinaryDataConverter
 ----
 
-By default, the `payload` column value (the Avro data) is the only message value.
-Configuration of `BinaryDataConverter` as the value converter propagates the `payload` column value as-is into the Kafka message value.
+By default, the `payload` column value (the Avro data) is the only message value. When storing data in Avro format the column must be of binary format (i.e. `bytea` in PostgreSQL),
+and value converter for the SMT must be `BinaryDataConverter`, which will propagate the `payload` column binary value as-is into the Kafka message value.
```

Comment:
```suggestion
The value converter for the SMT must also be set to `BinaryDataConverter`, so that it propagates the binary value of the `payload` column as-is into the Kafka message value.
```